sql/core/src/test/scala/org/apache/spark/sql/SQLQuerySuite.scala

```diff
@@ -1069,6 +1069,13 @@ class SQLQuerySuite extends QueryTest with SharedSparkSession with AdaptiveSpark
     spark.sessionState.conf.clear()
   }
 
+  test("Support retrieve hadoop configuration via set command") {
+    val key = "hadoop.this.is.a.test.key"
+    val value = "2018-11-17 13:33:33.333"
+    spark.sharedState.hadoopConf.set(key, value)
```

Comment:
Can we test with a valid use case? This doesn't seem to be valid.

Reply:
OK, I will add check for keys in `src/test/resources/hive-site.xml`